bin/common.sh

```diff
@@ -187,9 +187,42 @@ function randomPassword {
   date +%s | sha256sum | base64 | head -c 32 ; echo
 }
 
+function disable_sa_token_automount {
+  local ns sa_name should_disable
+  ns=$1
+  sa_name=$2
+  should_disable=${SEC_DISABLE_SA_TOKEN_AUTOMOUNT:-true}
+
+  if [ "$should_disable" == "true" ]; then
+     log_debug "Disabling automount of API tokens for serviceAccount [$ns/$sa_name]"
+     kubectl -n $ns patch serviceAccount $sa_name -p '{"automountServiceAccountToken":false}'
+  fi
+}
+
+function patch_pod_token_automount {
```

Comment:
Rename function to "enable_pod_token_automount"?  That would make it clear that it is enabling something?

Reply:
Done